packages/eslint-plugin-wpcalypso/docs/rules/no-unsafe-wp-apis.md

```diff
@@ -0,0 +1,45 @@
+<!-- eslint-disable wpcalypso/no-unsafe-wp-apis -->
```

Comment:
Documentation suggests this should disable the MD linter here but it's still reporting violations in this file in the forbid section 🤦 

@scinos am I doing it wrong? Do I need to use an .eslintrc?

Reply:
Sorry, the docs is not clear:

`<!-- eslint-disable wpcalypso/no-unsafe-wp-apis -->` Will disable the a __Markdown__ rule, it doesn't apply to code blocks.

To disable a rule inside a codeblock, you have to use the regular eslint syntax

```js
/* eslint-disable wpcalypso/no-unsafe-wp-apis */
import { __experimentalFeature } from '@wordpress/foo';
import { __unstableFeature } from '@wordpress/bar';
```